External/opencv/opencv-include.cmake

```diff
@@ -0,0 +1,22 @@
+
+set(OpenCV_STATIC (NOT BUILD_SHARED_LIBS))
```

Comment:
OpenCV was a huge pain. On windows I had no idea how to use the correct version when switching from release to debug builds. The work here loads the release version and creates duplicate targets for the debug targets so Visual Studio can compile both release and debug apps without needing to reconfigure CMake

Reply:
@UebelAndre The code dependencies of OpenCV is been phased out.

Reply:
@UebelAndre  Will be removing dependencies on this with subsequent updates.